quarkus-extension/engine/deployment/src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-quarkus.arc.auto-inject-fields=false
\ No newline at end of file
+quarkus.arc.auto-inject-fields=false
```

Comment:
We should remove this property now, since the default behavior will remain.
```suggestion
```

Reply:
👍